blocks/links-bar-block/features/links-bar/default.jsx

```diff
@@ -33,23 +32,34 @@ const LinksBar = ({ customFields: { navigationConfig = {}, ariaLabel } }) => {
 
 	return (
 		<>
-			<nav
+			<Stack
+				className={BLOCK_CLASS_NAME}
+				justification="center"
+				direction="row"
+				as="nav"
 				key={id}
-				className="links-bar"
 				aria-label={ariaLabel || phrases.t("links-bar-block.element-aria-label")}
 			>
 				{menuItems &&
 					menuItems.map((item, index) => (
-						<PrimaryFont as="span" className="links-menu" key={item._id}>
+						<React.Fragment key={item._id}>
 							{item.node_type === "link" ? (
-								<Link href={item.url} name={item.display_name} />
+								<Link href={item.url} className={`${BLOCK_CLASS_NAME}__menu-link`}>
+									{item.display_name}
+								</Link>
 							) : (
-								<Link href={item._id} name={item.name} />
+								<Link href={item._id} className={`${BLOCK_CLASS_NAME}__menu-link`}>
+									{item.name}
+								</Link>
 							)}
-							{content.children.length !== index + 1 && showSeparator ? "\u00a0 • \u00a0" : ""}
-						</PrimaryFont>
+							{content.children.length !== index + 1 && showSeparator ? (
+								<span className={`${BLOCK_CLASS_NAME}__menu-link-seperator`}>
```

Comment:
We actually have a Separator component that can be used here, and it's styling can be defined like the Link above.

Reply:
<Separator /> returns a span already.